mpw7 added

vijayank88 opened the pull request:
Signed-off-by: vijayank88 <paruthi143@gmail.com>

Following MPW7 designs added:
```
miranda_fpga
mpw7_microwatt
rioschip
riscduino_S4
rocket_alpha
waveform_generator
```

tspyrou reviewed (dismissed):
When the last CI turns green go ahead and merge.

vijayank88 commented:
`production_large-1` failed for `IIC_AudioDAC` which passed in previous CI. Fixed the issue with latest commit.
`patmos_chip` taking 14Hrs to complete flow. So skipping to save runtime and all mpw7 designs added mean will enable and keep part of CI again.
Added following MPW designs part of CI:
```
biquad_mpw7
figaro_mpw7
marmot_asic_v2
rvcore_chip1
rvj1-caravel-soc_mpw7
```
